.github/ISSUE_TEMPLATE/lsp_bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Language server client bug report
+description: Report a built-in lsp problem in Neovim
+labels: [bug, lsp]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Before reporting: search [existing issues](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and check the [FAQ](https://github.com/neovim/neovim/wiki/FAQ). Usage questions such as "How do I...?" or "Why isn't X language server/feature working?" belong on the [Neovim Discourse](https://neovim.discourse.group/c/7-category/7) and will be closed.
+
+  - type: input
+    attributes:
+      label: "Neovim Version"
+      description: "`nvim --version`:"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "Language server name/version:"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "Operating system/version:"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Checkhealth
+      description: |
+        Paste the results from `nvim -c ":checkhealth nvim lspconfig"` here:
+      render: markdown
+
+  - type: markdown
+    attributes:
+      value: |
+        Note, if the issue is with an autocompletion or other LSP plugin, please report to the upstream tracker.  Download the minmal config with `wget https://raw.githubusercontent.com/neovim/nvim-lspconfig/master/test/minimal_init.lua` and modify it to include any specific commands or servers pertaining to your issues.
+  - type: textarea
+    attributes:
+      label: "Reproduce"
+      description: |
+        Steps to reproduce using `nvim -u minimal_init.lua`:
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: "Expected behavior"
+  - type: textarea
+    attributes:
+      label: "Actual behavior"
+      description: "Logs, images and videos alongside the description is not necessary, but certainly appreciated."
+
+  - type: markdown
+    attributes:
+      value: |
+        Please paste the lsp log before and after the problem in a [secret gist](https://gist.github.com/). Paste the url to the gist in the text field below.
+
+        You can set log level like this.
+        `:lua vim.lsp.set_log_level("debug")`
+
+        You can find the location of the log with the following command.
+        `:lua print(vim.lsp.get_log_path())`
+
+  - type: input
+    attributes:
+      label: Lsp log url
```

Comment:
Do you mean "lsp url"  ?

Reply:
I did mean "Lsp log url" as in "the url to the lsp log", but I can change to "Lsp url".

Reply:
No, "Lsp url" is worse (do you meant the URL of the repo where I got my LSP server from?)

Better make it more explicit: "Link to your uploaded `lsp.log` file"